### ISSUE TITLE
MisterB's Lootable Things items only looted on 2nd attempt

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#131]
Nexus report

> I've noticed an oddity with this mod and MisterB's Lootable Things. That mod changes the mammoth skulls and hanging taproot sacks into containers that you can loot for mammoth tusks and taproots respectively. I'm noticing that Smart Harvest is showing signs of looting these containers (via both audio and the "glow briefly" feature), but inspecting the container shows that the item is still present. Smart Harvest will loot the mammoth skull or hanging taproot again after a few seconds, and will succeed on its second try. I've also inspected my inventory before and after this occurs, and I do -not- have extra items, so there is no item duplication glitch occurring.

**Comment by SteveTownsend** [comment]
I saw the same thing with a CONT-ified Mammoth Skull at NightgateInnExterior03 (just by the inn door)

**Comment by SteveTownsend** [comment]
I cannot get this to work without activating the item in script, which pops up the menu and is sort of beside the point . Making stubborn loot containers objects into a manual loot target so at least you get a message.

**Comment by SteveTownsend** [comment]
trying one more thing, it's a hack but this mod does not look to be universally used and the symptom only affects some of the CONTs

**Comment by SteveTownsend** [comment]
idk why these Containers don't behave like others. In any event scripting a silent normal activation (menu and all) is beyond me. I am going to just give the player the items they were not able to auto-loot. The REFR will be blacklisted at that point to avoid exploitation.

**Comment by SteveTownsend** [comment]
Xeroxing the container contents (auto-lootable portion) works fine. This is **the solution** for v1.10.0.0.

**Comment by SteveTownsend** [comment]
I have a work-round/fix for this under test in v1.10. I am focusing on the hanging taproots and mammoth skulls, but the change I made should work if any other rebadged containers cause the same issue.

I could not get those items to be auto-looted without activating them. Once I'd looked in the Container Menu, the autoloot fired OK. I spent all day yesterday trying to get the activation prerequisite scripted, but without success. The work round is to copy the lootable items if we go through a container and find looting failed outright on all valid targets. These containers are then blacklisted until game reload, to prevent an infinite supply of copied items being obtained.

I don't know why these Containers fail to autoloot, and I've spent long enough trying to get it working given this is a narrow compatibility issue. There is an exploit here where the container can be manually looted after the item copy happens. I also don't plan to worry about that.
